package.json

```diff
@@ -32,6 +32,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "styled-components": "5.3.5",
+    "subscriptions-transport-ws": "^0.11.0",
```

Comment:
Do we explicitly need to define this package? I don't think we import it anywhere, that's why I removed it before, same as in safe and wallet. It'd fine for me to define though, but the less we define, the less maintenance I'd say.